src/common/datomish/transact/bootstrap.cljc

```diff
@@ -38,6 +38,15 @@
                           :db/cardinality :db.cardinality/one}
    :db.alter/attribute   {:db/valueType   :db.type/ref
                           :db/cardinality :db.cardinality/many}
+
+   :db.schema/version    {:db/valueType   :db.type/long
+                          :db/cardinality :db.cardinality/one}
+
+   ;; unique-value because an attribute can only belong to a single
+   ;; schema fragment.
+   :db.schema/attribute  {:db/valueType   :db.type/ref
```

Comment:
`:db.type/ref` means that the range of the property is the space of all entities. The domain of every property is the space of all entities.

This is quite loosely modeled; there's no way in Datomish or Datomic to say "this property only applies to People, and can only point to Houses" — that's the realm of description logics.

Datomi{c,sh} are essentially RDF, which is a much weaker formalism.